peps/pep-0639.rst

```diff
@@ -685,8 +753,50 @@ they create are legally distributable, build tools SHOULD default to
 including at least the license files matching the above patterns, unless the
 user has explicitly specified their own.
 
+Examples of valid license files declaration:
+
+.. code-block:: toml
+
+    [project]
+    license-files = { globs = ["LICEN[CS]E*", "AUTHORS*"] }
+
+    [project]
+    license-files = { paths = ["licenses/LICENSE.MIT", "licenses/LICENSE.CC0"] }
+
+    [project]
+    license-files = { paths = [] }
+
+    [project]
+    license-files = { globs = [] }
+
+Examples of invalid license files declaration:
+
+.. code-block:: toml
+
+    [project]
+    license-files.globs = ["LICEN[CS]E*", "AUTHORS*"]
+    license-file.paths = ["LICENSE.MIT"]
```

Comment:
```suggestion
    license-files.paths = ["LICENSE.MIT"]
```
Also, it's confusing that none of the "valid" examples above use the dotted syntax. That made me think that there was something invalid about the dotted syntax.

Reply:
Good point, will rework that.